modules/service/src/main/scala/lucuma/odb/service/GuideEnvironmentService.scala

```diff
@@ -251,15 +252,18 @@ object GuideEnvironmentService {
           offset
         }
 
-      def getOffsets(
+      def getTimeAndOffsets(
         pid: Program.Id,
         oid: Observation.Id
-      ): F[Either[Error, Option[NonEmptyList[Offset]]]] =
+      ): F[Either[Error, (TimeSpan, Option[NonEmptyList[Offset]])]] =
         generator(commitHash, itcClient, plannedTimeCalculator)
           .digest(pid, oid)
           .map {
             _.leftMap(Error.GeneratorError(_))
-              .map { d => NonEmptyList.fromFoldable(d.science.offsets.union(d.acquisition.offsets)) }
+              .map { d => 
+                (d.fullPlannedTime.sum, 
```

Comment:
@swalker2m Is this the correct time to use for the duration for the calculation? Or should it be something like `digest.science.plannedTime.sum`?